hcatalog/server-extensions/src/main/java/org/apache/hive/hcatalog/listener/DbNotificationListener.java

```diff
@@ -155,9 +156,11 @@ public void onConfigChange(ConfigChangeEvent tableEvent) throws MetaException {
   @Override
   public void onCreateTable(CreateTableEvent tableEvent) throws MetaException {
     Table t = tableEvent.getTable();
+    FileIterator fileIter = t.getTableType().equals(TableType.EXTERNAL_TABLE.toString())
+                              ? null : new FileIterator(t.getSd().getLocation());
```

Comment:
if external table dump is enabled then we should allow file listing or do something to get the file list during dump

Reply:
i think its looks odd that create table will not dump the files but insert will ..i think we should stop both or allow both

Reply:
ok, will remove file listing even for inserts.